src/app/login/services/feedback.service.ts

```diff
@@ -1,9 +1,20 @@
 import { Injectable } from '@angular/core';
 
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { environment } from '../../../environments/environment';
+import { Feedback } from '../models/feedback.model';
+
 @Injectable({
   providedIn: 'root'
 })
 export class FeedbackService {
 
-  constructor() { }
+  constructor(private http: HttpClient) {
+  }
+
+  sendFeedback(feedback: Feedback): Observable<HttpResponse<any>> {
```

Comment:
I think this method "deserves" a unit test testing that calling sendFeedback makes a POST call under the hood